hazelcast-jclouds/src/main/java/com/hazelcast/jclouds/ComputeServiceBuilder.java

```diff
@@ -144,7 +146,6 @@ ComputeService build() {
         final String identity = getOrNull(JCloudsProperties.IDENTITY);
         String credential = getOrNull(JCloudsProperties.CREDENTIAL);
         final String credentialPath = getOrNull(JCloudsProperties.CREDENTIAL_PATH);
-
         checkNotNull(cloudProvider, "Cloud provider is not set");
         checkNotNull(identity, "Cloud provider identity is not set");
```

Comment:
what if user only set `iam-role` ... in that case , user doesn't have to set `identity`. right?